src/services/measure/measurearea.js

```diff
@@ -0,0 +1,84 @@
+goog.provide('ngeo.interaction.MeasureArea');
+
+goog.require('ngeo.interaction.Measure');
+goog.require('ol.geom.Polygon');
+goog.require('ol.interaction.Draw');
+
+
+
+/**
+ * @classdesc
+ * Interaction dedicated to measure length.
+ *
+ * @constructor
+ * @extends {ngeo.interaction.Measure}
+ * @param {{
+ *    startMsg: (string|undefined),
+ *    continueMsg: (string|undefined),
+ *    style: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *    sketchStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined)
+ * }=} opt_options Options
```

Comment:
We need to create an `ngeox.js` externs file for the interaction options, as `olx.js` in ol3.